dispatch/route_test.go

```diff
@@ -853,3 +853,78 @@ routes:
 		}
 	}
 }
+
+func TestRouteID(t *testing.T) {
```

Comment:
if we're only testing the top-level routes, there's no need for having sub-routes?
also it might be good to have routes with same key but different ids?

Reply:
"I'm missing the context about this change. Is it linked to the pagination API?"

-> It is related as we need to UUID an aggregation group, aggregation group can be uuid by group_labels + routeId

Reply:
=> if we're only testing the top-level routes, there's no need for having sub-routes?
also it might be good to have routes with same key but different ids?

Added the test